### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by ghostwords**
Following up on #1468.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ can help. Here's a quick orientation to guide your adventure:
   translation pull request. See the [translations
   label](https://github.com/EFForg/privacybadger/issues?utf8=%E2%9C%93&q=label%3Atranslations%20)
   for related issues and more example pull requests.
-* Tests are crucial, so here's a [*guide*](/doc/tests.md) on how to use them.
+* Tests are crucial, so here's a [guide](/doc/tests.md) on how to use them.
```
Comment (cowlicks) [inline .github/CONTRIBUTING.md]: I actually meant for this to be bold, not italicized, to add emphasis.

File: doc/CONTRIBUTION-GUIDELINES.md [hunk 2]
```diff
@@ -1,53 +0,0 @@
-# PRIVACY BADGER CONTRIBUTION GUIDELINES
```
Comment (cowlicks) [inline doc/CONTRIBUTION-GUIDELINES.md]: 👍

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (ghostwords) [inline .github/CONTRIBUTING.md]: Ah, but I'm not sure this link deserves to stand out over other links on the page.

Comment (cowlicks) [inline .github/CONTRIBUTING.md]: I think guides are much more frequently read, and more useful, so I think they should easy to find. The fixing broken sites link above should be bolded too imo.

File: README.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ And of course, we try to respond in a timely manner to GitHub issues, pull reque
 ## Developing
 
 We're glad you want to help! We have a **[handy guide for
-contributors](/doc/CONTRIBUTING.md)**. It can hopefully point you to helpful
+contributors](/.github/CONTRIBUTING.md)**. It can hopefully point you to helpful
```
Comment (cowlicks) [inline README.md]: Would you be friendly to moving it to the root instead? That directory is getting crowded, but I think this guide is useful enough to go there. I'd like it to be easily noticed by people skimming the project.

**Pull request merged**